app/views/welcome/foods.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Your query found <%= @foods.total_hits %> results for <%= @q %></h1>
+
+<% @foods.foods(10).each do |food| %>
```

Comment:
The naming here could use some works to differentiate what each object is.

Reply:
Also, you only made 10 food objects within your Foods class, so you don't need to scope this to the first 10 again in the view.